library/Icingadb/ProvidedHook/Reporting/SlaReport.php

```diff
@@ -234,7 +237,21 @@ public function getHtml(Timerange $timerange, array $config = null)
                 $slaClass = 'ok';
             }
 
-            $cells[] = Html::tag('td', ['class' => "sla-column $slaClass"], round($sla, $precision));
+            if ($config['breakdown'] === 'hour' && $slaClass === 'nok') {
+                foreach ($row->getDimensions() as $dimension) {
+                    $cells[] = Html::tag('td', null, $dimension);
+                }
+            } elseif ($config['breakdown'] != 'hour') {
```

Comment:
Think about how you can handle these `if ... elseif` branches in just a single `if` block.